packages/auth/src/const.ts

```diff
@@ -11,15 +11,17 @@ export const ROLE_NAME = {
   Guest: "guest",
   BaseUser: "base_user",
   Applicant: "applicant",
+  Manager: "manager",
   PoolOperator: "pool_operator", // TODO: remove during Communities cleanup, ticket #10368
   RequestResponder: "request_responder", // TODO: remove during Communities cleanup, ticket #10368
   CommunityManager: "community_manager", // TODO: remove during Communities cleanup, ticket #10368
   ProcessOperator: "process_operator",
   CommunityRecruiter: "community_recruiter",
   CommunityAdmin: "community_admin",
   PlatformAdmin: "platform_admin",
-  Manager: "manager",
 } as const;
 
 type ObjectValues<T> = T[keyof T];
 export type RoleName = ObjectValues<typeof ROLE_NAME>;
+
+export const NAV_ROLE = "nav_role";
```

Comment:
A bit more descriptive:
```suggestion
export const NAV_ROLE_STORAGE_KEY = "nav_role";
```

Reply:
Nice call :+1: Fixed in [c0a75d8](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11501/commits/c0a75d801f67b24a7a4ba7e113ff1ee664c2e0bd)